src/main/java/seedu/address/logic/parser/BookShelfParser.java

```diff
@@ -149,10 +149,10 @@ private Command getCommand(String commandWord, String arguments) throws ParseExc
 
         case ListCommand.COMMAND_WORD:
             return new ListCommandParser().parse(arguments);
-
+        //@@author 592363789
```

Comment:
Remove your changes to this file. Don't collate this kind of small things.

Reply:
How to remove your changes to the generated `.md` files of others?